drivers/usbdev/usbdev_trprintf.c

```diff
@@ -97,10 +97,10 @@ void usbtrace_trprintf(trprintf_t trprintf, uint16_t event, uint16_t value)
     case TRACE_DEVINIT:
 #ifdef CONFIG_USBDEV_TRACE_STRINGS
       trprintf("%-18s   : %-40s %04x\n", "DCD initialize",
-               TRACE_DATA(event), "Initialized", value);
+               "Initialized", value);
 #else
       trprintf("%-18s   : %04x\n", "DCD initialize",
-               TRACE_DATA(event), value);
```

Comment:
What happened here? Does TRACE_DEVINIT event not need a TRACE_DATA code be output via some %3d here, like in the other events? Or was this someone's copy-paste error that the PR unearthed? git annotate shows a 9 year old commit touching there lines.

Reply:
Seems to be some copy-paste leftover as `TRACE_DATA` is used only in `default` case if this switch

Reply:
As @pkarashchenko said, TRACE_DATA is used for other event but not initialization.